test/binary-elementwise-operator-tester.cc

```diff
@@ -723,6 +724,151 @@ void BinaryElementwiseOperatorTester::TestF16() const {
   }
 }
 
+void BinaryElementwiseOperatorTester::TestS32() const {
+  ASSERT_NE(operation_type(), OperationType::Unknown);
+  ASSERT_LT(qmin(), qmax());
+
+  xnnpack::ReplicableRandomDevice rng;
+  std::uniform_int_distribution<int32_t> s32dist(std::numeric_limits<int32_t>::min(),
```

Comment:
min is zero, not negative: https://en.cppreference.com/w/cpp/types/numeric_limits/min